### ISSUE TITLE
Zwei Anlagen

### ISSUE THREAD

**Issue by iobroker99** [thovid/ioBroker.viessmannapi#58]
Hallo,
ich betreibe 2 Viessmann Anlagen an 2 Standorten, beide im gleichen ViCare Account. Die eine Anlage sehe ich im Adapter, die andere nicht. Gibts einen Trick?

**Comment by thovid** [comment]
Hi, im Moment unterstützt der Adapter zwei Anlagen mit dem selben Account nicht. Ich nehme das in die Liste zukünftiger features auf.

**Comment by iobroker99** [comment]
Super, Vielen Dank. Das würde mir sehr helfen, denn leider sehe ich im Adapter ausgerechnet die Anlage die ich steuern möchte nicht ;-)

**Comment by nightwrit** [comment]
Das wäre prima. Ich habe hier eine Hybrid Anlage bestehend aus einer Vitocal 200S und einem Vitocrossal

**Comment by TA2k** [comment]
@nightwrit @iobroker99  könnt ihr checken ob das mit funktioniert

https://forum.iobroker.net/topic/46778/test-viessmannapi-v2-0-0/

**Comment by iobroker99** [comment]
nein, funktioniert leider nicht. Habe das mal als Issue aufgemacht
Ansonsten sieht das sehr gut aus, Vielen Dank für die Arbeit. Leider sehe ich aber genau die Anlage die ich brauche nicht ;-))
